packages/smithy-http/src/smithy_http/serializers.py

```diff
@@ -328,7 +328,7 @@ def __init__(
             :py:class:`HTTPHeaderTrait` will be checked instead. Required when
             collecting list entries.
         :param headers: An optional list of header tuples to append to. If not
-            set, one will be created.
+            set, one will be created. Values appended will not be escaped.
```

Comment:
What is handling any required escaping now?

Reply:
They're escaped when they're joined